MobileCenterAnalytics/MobileCenterAnalytics/Support/MobileCenterAnalytics.xcconfig

```diff
@@ -1,3 +1,3 @@
 #include "../../../Global.xcconfig"
 
-OTHER_LDFLAGS = $(inherited) -framework Foundation -framework UIKit -framework CoreTelephony;
```

Comment:
If we want to build *framework target, then build shows link errors related to SQLite library